src/libraries/Authentication/Authentication.Msal/MsalAuth.cs

```diff
@@ -75,12 +75,12 @@ public async Task<string> GetAccessTokenAsync(string resourceUrl, IList<string>
 
             // Get or create existing token. 
             _cacheList ??= new ConcurrentDictionary<Uri, ExecuteAuthenticationResults>();
-            if (_cacheList.ContainsKey(instanceUri))
+            if (_cacheList.TryGetValue(instanceUri, out ExecuteAuthenticationResults value))
             {
                 if (!forceRefresh)
                 {
-                    var accessToken = _cacheList[instanceUri].MsalAuthResult.AccessToken;
-                    var tokenExpiresOn = _cacheList[instanceUri].MsalAuthResult.ExpiresOn;
+                    var accessToken = value.MsalAuthResult.AccessToken;
```

Comment:
Can you alter the suggestion there to use a name other than Value (makes my head hurt b/c its a context verb in C# )